cinema/views.py

```diff
@@ -167,6 +179,24 @@
 
         return MovieSessionSerializer
 
+    @extend_schema(
+        parameters=[
+            OpenApiParameter(
+                name="date",
+                type=OpenApiTypes.DATE,
+                description="Filter movie sessions by date",
+            ),
+            OpenApiParameter(
+                name="movie",
+                type=OpenApiTypes.INT,
+                description="Filter movie sessions by movie",
+            ),
+        ]
+    )
+    def list(self, request, *args, **kwargs):
+        """ Get a list of all available buses """
```

Comment:
Similar to the `MovieViewSet`, the docstring for the `list` method in `MovieSessionViewSet` incorrectly mentions 'buses'. It should be updated to reflect that it returns a list of movie sessions.